### PR TITLE
:zap: Mt 1215 dialogbox fix

### PR DESCRIPTION
**Pull request by miloszhygen**

### CODE REVIEW

File: src/svelte/DialogBox.svelte [hunk 1]
```diff
@@ -19,16 +20,21 @@
...
+  <div class="" aria-flowto={`${dialogBoxHeadingId} ${dialogCloseButton}`}>
     <button
+      id={dialogCloseButton}
       data-testid="dialog-box-close"
       class="button button--link dialog-box--button"
       on:click={handleClose}
       aria-label={closeBtnAriaLabel} />
-  </h2>
+    {#if title}
+      <h2 id={dialogBoxHeadingId} class="h4 dialog-box--title" aria-label={title}>
```
Comment (lha047) [inline src/svelte/DialogBox.svelte:33]: Dont need aria-label on a heading?

File: src/svelte/DialogBox.svelte [hunk 2]
```diff
@@ -19,16 +20,21 @@
   data-testid="dialog-box"
   role="dialog"
   tabindex="0"
-  aria-labelledby={dialogBoxHeadingId}
+  aria-label="dialog boks"
```
Comment (lha047) [inline src/svelte/DialogBox.svelte:23]: This cant be hard coded

**Review by lha047 — CHANGES_REQUESTED** [review]
See comments

**Review by itemodahl — APPROVED** [review]
Awesome job!

**Pull request merged**